content/06.discussion.md

```diff
@@ -54,6 +54,34 @@ In assessing the upper bound on the predictive performance of a deep learning mo
+
+#### Bayesian hypernetworks
+
+Another framework for approximate Bayesian inference in deep learning is utilizing Bayesian hypernetworks [@tag:bayesian_hypernets]. A hypernetwork is a network that learns to map its input to the parameters of another network, the primary network. Both the hypernetwork and the primary network are trained together as a single model. Krueger et al introduced Bayesian hypernetworks and used an invertible generative network to map isotropic Gaussian noise to parameters of the primary network. Using an invertible hypernetwork allowed computationally cheap sampling and tractable estimation of the posterior. Bayesian hypernetworks are better at dealing with multimodal posterior distributions than other Bayesian deep learning approaches.
+
+#### Other approaches
+
+Several other techniques have been proposed for effectively estimating predictive uncertainty as uncertainty quantification for neural networks continues to be an active research area. Recently, McClure and Kriegeskorte observed that test-time sampling improved calibration of the probabilistic predictions, sampling weights led to more robust uncertainty estimates than sampling units, and spike-and-slab sampling is superior to Gaussian dropconnect and Bernoulli dropout [@tag:mcclure_bayesian]. Meanwhile, Lakshminarayanan et al proposed using deep ensembles, which are traditionally used for boosting predictive performance, on standard (non-Bayesian) neural networks to obtain well-calibrated uncertainty estimates that are comparable to those obtained by Bayesian neural networks [@tag:uncertainty_ensembles]. In cases where model uncertainty is known to be caused by a difference in training and testing distributions, domain adaptation based techniques can help mitigate the problem [@tag:domain_adapt_uncertainty].
+
+#### Future work
+
+Despite the success and popularity of deep learning, some deep learning models can be surprisingly brittle. Researchers are actively working on modifications to deep learning frameworks to enable them to handle probability and embrace uncertainty. Most notably, Bayesian modeling and deep learning are being integrated with renewed enthusiasm. As a result, several opportunities for innovation arise: understanding the causes of model uncertainty can lead to novel optimization and regularization techniques, assessing the utility of uncertainty estimation techniques on various model architectures and structures can be very useful to practitioners, and extending Bayesian deep learning to unsupervised settings can be a significant breakthrough [@tag:gal_thesis]. Unfortunately, uncertainty quantification techniques are underutilized in the computational biology communities and largely ignored in the current deep learning for biomedicine literature. Thus, the practical value of uncertainty quantification in biomedical domains is yet to be demonstrated.
```

Comment:
Don't like this last sentence - you've established above why you need uncertainty qualification, but here you say the practical value is yet to be demonstrated, Just drop it or replace with sentence saying UQ is yet to be appreciated or widely used?

Reply:
☝️ I agree with this. Prefer "yet to be appreciated or widely used" than a total drop, but I'd be fine with either.

Reply:
I changed the last sentence.